migrations/900000000000029_sqlq_support_schema.up.sql

```diff
@@ -0,0 +1,40 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS mergestat.job_types (
+    id uuid PRIMARY KEY DEFAULT public.gen_random_uuid() NOT NULL,
+    name text NOT NULL,
+    queue text NOT NULL,
+    description text,
+    default_parameters jsonb,
+    priority int,
+    schedule text,
```

Comment:
with @patrickdevivo we discuss that we would like this to be a cron string instead of a int or timestamp , that way we would have more freedom to manipulate the way we handles schedules